### PR TITLE
Update Berks and ChefDK

### PR DESCRIPTION
**Pull request by tas50**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -305,9 +305,9 @@ See [here](https://github.com/zuazo/docker-in-travis#acknowledgements) for more.
     Licensed under the Apache License, Version 2.0 (the "License");
     you may not use this file except in compliance with the License.
     You may obtain a copy of the License at
-    
+
         http://www.apache.org/licenses/LICENSE-2.0
-    
+
```
Comment (zuazo) [inline README.md:310]: Please, don't remove those spaces. That will break the license box.

**Comment by zuazo** [comment]
@tas50 thanks for the contribution!! :smiley:

File: README.md (hunk 1 above)
Comment (zuazo) [inline README.md:310]: OK. Doesn't seem to break the box on GitHub. Sorry. My fault  :disappointed:

Comment (tas50) [inline README.md:310]: That's just my editor auto correcting but I'll avoid any of those in the future

**Pull request merged**